src/main/java/com/itacademy/cms/exeption/CategoryNotFoundException.java

```diff
@@ -1,6 +1,6 @@
 package com.itacademy.cms.exeption;
 
-public class CategoryNotFoundException extends RuntimeException {
+public class CategoryNotFoundException extends Exception {
```

Comment:
Please change Exception to RuntimeException

Reply:
i thing that all exceptions must be the same type, so i did all of them the same type